sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DefaultDocumentQueryExecutionContext.java

```diff
@@ -257,8 +257,12 @@ private static boolean isClientSideContinuationToken(String continuationToken) {
                 return true;
             }
 
-            ValueHolder<TakeContinuationToken> outTakeContinuationToken = new ValueHolder<>();
-            return TakeContinuationToken.tryParse(continuationToken, outTakeContinuationToken);
+            ValueHolder<TopContinuationToken> outTopContinuationToken = new ValueHolder<>();
+            if (TopContinuationToken.tryParse(continuationToken, outTopContinuationToken)) {
+                return true;
+            }
+            ValueHolder<LimitContinuationToken> outLimitContinuationToken = new ValueHolder<>();
```

Comment:
I dont think we use this path anymore :-)! There are various other composite continuation tokens which could mark the continuation token as clientSideContinuationTokens.